<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>591</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">Codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descricao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Taxa Finan. R$</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Taxa ADM R$</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cobrar no Pedido</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(132, 168, 163);
color: rgb(33, 25, 255);
font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Financeiras Cadastradas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_inclusao">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Inclusao</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_alteracao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Alteracao</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_consulta_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exclusao</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incl_aliquota">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Inclusao de Aliquotas</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
